Bigquery - Support for microbatch execution (#9621)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/core-upgrade/03-upgrading-to-v1.12.md

    By default, dbt sets `scheduler` to `DISABLE` (unlike Snowflake’s native default of `ENABLE`), so dbt controls the refresh schedule unless you explicitly opt in to Snowflake’s scheduler. If you specify `target_lag` without setting `scheduler`, dbt automatically sets `scheduler` to `ENABLE`.
- You can use the [`snowflake_initialization_warehouse`](/reference/resource-configs/snowflake-configs#initialization-warehouse) parameter to specify a warehouse for the initial build and reinitialization of a dynamic table, separate from `snowflake_warehouse` which is used for regular incremental refreshes.
- You can create Snowflake dynamic tables as transient (no [Fail-safe period](https://docs.snowflake.com/en/user-guide/data-failsafe)) by setting the [`transient`](/reference/resource-configs/snowflake-configs#transient-dynamic-tables) config on models. 

    When `transient` is not set on a model, the [`snowflake_default_transient_dynamic_tables`](/reference/global-configs/snowflake-changes#the-snowflake_default_transient_dynamic_tables-flag) flag controls the default. Set this flag to `true` to make all dynamic tables transient by default.

### BigQuery

> - BigQuery now supports [parallel microbatch execution](/docs/build/parallel-batch-execution). dbt automatically detects whether batches can run concurrently for BigQuery microbatch models.

Author: This is the only new line on this file, please review this. Thanks!

> - BigQuery now supports [parallel microbatch execution](/docs/build/parallel-batch-execution). dbt automatically detects whether batches can run concurrently for BigQuery microbatch models.

Reviewer: Hiya Bianca

On the BigQuery upgrade bullet: "dbt automatically detects whether batches can run concurrently for BigQuery microbatch models" and under the BigQuery heading it might read like auto-detection is new/BigQuery-only. 

So maybe something like:

"BigQuery now supports [parallel microbatch execution](https://docs.getdbt.com/docs/build/parallel-batch-execution?version=2.0) in addition to Snowflake".

Author: ```suggestion
- BigQuery now supports [parallel microbatch execution](/docs/build/parallel-batch-execution), in addition to Snowflake.
```
